README.md

```diff
@@ -0,0 +1,64 @@
+# Diplomat
```

Comment:
It would be nice to link to a pre-print of our work if somebody wants more details on the design considerations. I'll ask Jim about this

Reply:
Yeah, we can add it later!